install.sh

```diff
@@ -502,6 +506,18 @@ function open_tensorleap_url() {
 
 function main() {
   echo Please note that during the installation you may be required to provide your computer password to enable communication with the docker.
+  init_var_dir
+  log "starting run"
+  log "DISABLE_DOCKER_CHECKS = $DISABLE_DOCKER_CHECKS"
+  log "DISABLE_REPORTING = $DISABLE_REPORTING"
+  log "DISABLE_CLUSTER_CREATION = $DISABLE_CLUSTER_CREATION"
+  log "FILES_BRANCH = $FILES_BRANCH"
+  log "DATA_VOLUME = $DATA_VOLUME"
+  log "INSTALL_ID = $INSTALL_ID"
+  log "USE_LOCAL_HELM = $USE_LOCAL_HELM"
+  log "USE_GPU = $USE_GPU"
+  log "INSECURE = $INSECURE"
```

Comment:
Consider to extract all those to another function like `log_environment_variables`

Reply:
done